projects/power_select/inc/power_select_defs.h

```diff
@@ -26,22 +27,31 @@
 #define POWER_SELECT_AUX_MAX_CURRENT_MA 8220
 #define POWER_SELECT_AUX_MAX_VOLTAGE_MV 15820
 
+// Cell needs to be above 2V to work, so setting cutoff at 2100 mV
+#define POWER_SELECT_CELL_MIN_VOLTAGE_MV 2100
+
 // Offsets for use in fault bitset
 typedef enum {
-  POWER_SELECT_AUX_OVERVOLTAGE = 0,
-  POWER_SELECT_DCDC_OVERVOLTAGE,
-  POWER_SELECT_PWR_SUP_OVERVOLTAGE,
-  POWER_SELECT_AUX_OVERCURRENT,
-  POWER_SELECT_DCDC_OVERCURRENT,
-  POWER_SELECT_PWR_SUP_OVERCURRENT,
-  POWER_SELECT_DCDC_FAULT,
+  POWER_SELECT_FAULT_AUX_OV = 0,
+  POWER_SELECT_FAULT_DCDC_OV,
+  POWER_SELECT_FAULT_PWR_SUP_OV,
+  POWER_SELECT_FAULT_AUX_OC,
+  POWER_SELECT_FAULT_DCDC_OC,
+  POWER_SELECT_FAULT_PWR_SUP_OC,
+  POWER_SELECT_FAULT_DCDC_PIN,
   NUM_POWER_SELECT_FAULTS,
 } PowerSelectFault;
 
+// Offsets for use in warning bitset
+typedef enum {
+  POWER_SELECT_WARNING_BAT_LOW = 0,
+  NUM_POWER_SELECT_WARNINGS,
+} PowerSelectWarning;
+
 // Faults that require the LTC to be turned off
 // See https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/1055326209/Power+Selector+Board

```

Comment:
Sorry, didn't see this until now haha. Makes sense, I realized I don't have a Confluence doc for power select specifically, should I just make one similar to [this one](https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/2219114569/Decoding+Power+Distribution+Error+Codes) for them?

Was planning on updating the [driver display errors page](https://uwmidsun.atlassian.net/wiki/spaces/S/pages/2347598112/Errors+and+severities) as well, but after this gets finalized since Ryan wanted me to double-check with Kristen whether the cell low warning fits there.

I can just do these alongside this ticket if that's easier, it should be pretty short since the errors aren't complicated.